Documentation/DSCResources/VMHostGraphics/VMHostGraphics.md

```diff
@@ -0,0 +1,100 @@
+# VMHostGraphics
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Name** | Key | string | Name of the VMHost to configure. ||
+| **Credential** | Mandatory | PSCredential | Credentials needed for connection to the specified Server. ||
+| **DefaultGraphicsType** | Mandatory | GraphicsType | The default graphics type for the specified VMHost. This default value is overridden if an individual device graphics type is specified. If the specified VMHost supports a single graphics type, specifying an individual graphics device is optional. | Shared, SharedDirect |
+| **SharedPassthruAssignmentPolicy** | Mandatory | SharedPassthruAssignmentPolicy | The policy for assigning shared passthrough VMs to a host graphics device. | Performance, Consolidation |
+| **DeviceId** | Optional | string | The Graphics device identifier (ex. PCI ID). ||
+| **DeviceGraphicsType** | Optional | GraphicsType | The graphics type for the specified Device in 'DeviceId' property. | Shared, SharedDirect, Unset |
+| **RestartTimeout** | Optional | int | The time in minutes to wait for the VMHost to restart before timing out and aborting the operation. The default value is 5 minutes. ||
+
+## Prerequisite
+The specified VMHost must be in **Maintenance** mode.
+
+## Description
+The resource is used to update the Graphics configuration by changing the DefaultGraphicsType and SharedPassthruAssignmentPolicy values. If a Graphics Device is specified it also changes its Graphics Type value. The resource also restarts the VMHost after a successful Update operation.
```

Comment:
As we discussed split to two resources one for Host configuration and another for Device configuration

Reply:
Done.